src/components/GameItem.js

```diff
@@ -13,17 +30,25 @@ class GameItem extends Component {
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type];
+
+    let spottedClass = ''
+    if (this.props.type === 'litter'){
+      spottedClass = 'spotted-litter';
+    } else {
+      spottedClass = 'spotted-nature'
+    }
+
+    const itemClass = this.state.isSpotted ? 'game-item '+ spottedClass : 'game-item';
 
     return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
+      <div className={itemClass} style={itemStyle}>
+        <img src={icon} alt="Item" className='icon-item' onClick={ this.onSpottedItem }></img>
```

Comment:
To get a better chance of the click registering, I would put the `onClick` in the div tag rather than the img tag.